README.md

```diff
@@ -17,43 +17,65 @@ You can use a stack image to do any of the following (and more):
 
 You can try a [relatively recent build of the jupyter/base-notebook image on mybinder.org](https://mybinder.org/v2/gh/jupyter/docker-stacks/master?filepath=README.ipynb)
 by simply clicking the preceding link.
-Otherwise, three examples below may help you get started if you [have Docker installed](https://docs.docker.com/install/),
+Otherwise, the three examples below may help you get started if you [have Docker installed](https://docs.docker.com/install/),
 know [which Docker image](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html) you want to use
 and want to launch a single Jupyter Server in a container.
 
 The [User Guide on ReadTheDocs](https://jupyter-docker-stacks.readthedocs.io/) describes additional uses and features in detail.
 
-**Example 1:** This command pulls the `jupyter/scipy-notebook` image tagged `b418b67c225b` from Docker Hub if it is not already present on the local host.
+**Example 1:**
+
+This command pulls the `jupyter/scipy-notebook` image tagged `b418b67c225b` from Docker Hub if it is not already present on the local host:
+
+```bash
+docker run -p 8888:8888 jupyter/scipy-notebook:b418b67c225b
+```
+
 It then starts a container running a Jupyter Server and exposes the server on host port 8888.
 The server logs appear in the terminal.
+
 Visiting `http://<hostname>:8888/?token=<token>` in a browser loads JupyterLab,
-where `hostname` is the name of the computer running docker and `token` is the secret token printed in the console.
+where:
+
+- `hostname` is the name of the computer running docker
+- `token` is the secret token printed in the console
+
 The container remains intact for restart after the Jupyter Server exits.
 
-```bash
-docker run -p 8888:8888 jupyter/scipy-notebook:b418b67c225b
-```
+**Example 2:**
```

Comment:
Let's actually get rid of example 1 completely, and explain in a separate bullet point, that we forward port 8888 of the container to port 10000 of the host machine and you can forward to the same port using `8888:8888`